### ISSUE TITLE
Topics section

### ISSUE THREAD

**Issue by schalkneethling** [mdn/developer-portal#1720]
The topics list should be marked up as an unordered list. There is also an incorrect heading hierarchy.

A suggested restructuring could look something like:

```
<section id="topics" aria-labelledby="topics-main-heading">
  <h2 id="topics-main-heading">Browse products &amp; technologies</h2>

  <ul class=" mzp-l-card-quarter">
    <li class="mzp-c-card mzp-c-card-extra-small">
      <a class="mzp-c-card-block-link" href="/topics/browser-extensions/">
        <h3 class="mzp-c-card-title"><span>Add-Ons</span></h3>
        ...
      </a>
    </li>
    <li class="mzp-c-card mzp-c-card-extra-small">
      <a class="mzp-c-card-block-link" href="/topics/browser-extensions/">
        ...
      </a>
    </li>
    ...
  </ul>
</section>
```

Not the change in heading elements in the above code snippet.

**Comment by stevejalim** [comment]
I think this will be automatically handled by #1719
